### PR TITLE
Add initial batch of documentation for CO2.js

### PR DESCRIPTION
**Pull request by fershad**
This PR adds the first set of developer documentation for the CO2.js library.

- [Overview](https://github.com/thegreenwebfoundation/docs-example/blob/co2js-overview/src/docs/co2js/overview.md)
- [Getting started: NodeJS](https://github.com/thegreenwebfoundation/docs-example/blob/co2js-overview/src/docs/co2js/tutorials/getting-started-node.md)
- [Methodologies for calculating website carbon emissions](https://github.com/thegreenwebfoundation/docs-example/blob/co2js-overview/src/docs/co2js/explainer/methodologies-for-calculating-website-carbon.md)

Documentation ready for review (typos, process, language etc.).

Changes can also be previewed in Gitpod using: https://gitpod.io/#https://github.com/thegreenwebfoundation/docs-example/tree/co2js-overview

### CODE REVIEW

**Comment by fershad** [comment]
[CO2.js Overview](https://trello.com/c/xjJHQZS7/13-co2js-overview)

**Comment by fershad** [comment]
[CO2.js - Getting started (server)](https://trello.com/c/eimLPaFI/14-co2js-getting-started-server)

**Comment by fershad** [comment]
[CO2.js - Methodologies for calculating website carbon](https://trello.com/c/KspdwAIr/16-co2js-methodologies-for-calculating-website-carbon)

File: package.json [hunk 1]
```diff
@@ -15,11 +15,13 @@
     "@11ty/eleventy-navigation": "^0.3.3",
     "@tailwindcss/typography": "^0.5.2",
     "autoprefixer": "^10.4.7",
+    "daisyui": "^2.15.3",
     "postcss": "^8.4.14",
     "tailwindcss": "^3.1.2",
-    "vite": "^2.9.12",
-    "daisyui": "^2.15.3"
+    "vite": "^2.9.12"
```
Comment (mrchrisadams) [inline package.json:21]: hey Fershad - what does vite do here?

I get that it is a bundler that uses esbuild under the hood, but I can't tell what it's doing in this PR, and I can't see it being explicitly called in any npm commands or config files.

If it's not actively being used, could we remove, and add it later with a bit of explanation about how to use it in development?

**Review by mrchrisadams — APPROVED** [review]
At my end I'm happy to merge this in, as long as we have another issue to make sure the compiled code is being deployed to http://developers.thegreenwebfoundation.org when future code is merged into main

File: package.json (hunk 1 above)
Comment (fershad) [reply package.json]: I think it came along for the ride when copying the DaisyUI getting started code for 11ty. I've removed it and don't see any ill effects.

**Comment by fershad** [comment]
Updated this branch with some further design changes. Ready to be merged into `main`.

**Pull request merged**